packages/cli-kit/src/public/node/node-package-manager.ts

```diff
@@ -521,3 +537,26 @@ async function getLatestNPMPackageVersion(name: string) {
   debug(content`Getting the latest version of NPM package: ${token.raw(name)}`)
   return latestVersion(name)
 }
+
+/**
+ * Writes the package.json file to the given directory.
+ *
+ * @param directory - Directory where the package.json file will be written.
+ * @param packageJSON - Package.json file to write.
+ */
+export async function writePackageJSON(directory: string, packageJSON: PackageJson): Promise<void> {
+  debug(content`JSON-encoding and writing content to package.json at ${token.path(directory)}...`)
+  const packagePath = join(directory, 'package.json')
+  await writeFile(packagePath, JSON.stringify(packageJSON, null, 2))
+}
+
+/**
+ * Updates the package.json file with the given name and sets the user to the current OS username.
+ *
+ * @param packageJSON - Package.json file to update.
+ * @param name - Name to set.
+ */
+export async function updateAppData(packageJSON: PackageJson, name: string): Promise<void> {
```

Comment:
The naming `updateAppData` is tied to app development. Also the function doesn't seem to provide much value. One could take the `packageJSON` instance and set the values directly. What about removing it?